scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -134,7 +134,7 @@ export class LaneShowCmd implements Command {
   alias = '';
   options = [
     ['j', 'json', 'show the lane details in json format'],
-    ['r', 'remote', 'show the lane from remote'],
+    ['r', 'remote', 'show the lane from remote'], // @david what does this mean?
```

Comment:
but if i have it locally, what will it show from the remote (as opposed to not using this flag)?

And if i try to show a remote lane (that i dont have locally) without this flag, what happens?

Reply:
> but if i have it locally, what will it show from the remote (as opposed to not using this flag)?

it'll show the versions of each components from the remote. it's possible that locally I'm behind/ahead/diverged

> And if i try to show a remote lane (that i dont have locally) without this flag, what happens?

It'll probably throw LaneNotFound, coz it can't find it locally

Reply:
btw, what happens if i try to run `bit lane main --remote`, or just `bit lane --remote` if i'm on the main lane?
won't that break something as main is everything?

Reply:
I guess you mean `bit lane show main --remote <renmote-name>`. It'll show the components on this scope that are available on main. (which practically means that either they were created on main or merged into main).

Reply:
on which scope? no scope has been mentioned here... and the --remote flag has been used so it's looking remotely, not locally
(and yep, that's the command i meant :)

Reply:
I was actually confused here. the `--remote` is boolean. So to get the "main" of a remote scope you can run `bit lane show your-scope/main --remote`.

Reply:
why is this limited to a specific scope? the main lane holds all components from all scopes no? And a lane isnt limited to storing only components of the scopes it's stored in.

(Another conceptual issue with having lanes stored on a scope, but not actually related to that scope)

Reply:
(i'm going to resolve this conversation as this discussion is beyond the scope here, and we can think whether it's worth opening separately)